src/util.h

```diff
@@ -115,6 +115,11 @@ bool Truncate(const string& path, size_t size, string* err);
 #endif
 
 #ifdef _WIN32
+/// UTF-8 converters
+std::wstring Utf8ToWide(const std::string& s);
+std::string WideToUtf8(const std::wstring& s);
+void convertCommandLine(int argc, wchar_t** wargv, char** argv);
```

Comment:
Why not return argv instead?

Reply:
Changed so that convertCommandLine return char** instead and updated string allocation.